nix/derivation.nix

```diff
@@ -24,7 +24,7 @@ with pkgs; stdenv.mkDerivation rec {
     # Static builds will use PKG_CONFIG_PATH approach instead
   ];
   prePatch = ''
-    export CFLAGS='-static -pthread'
+    export CFLAGS='-static'
```

Comment:
For the last 4 years this reference to `-pthread` has been there without the need for it?

If so, perhaps this is worthy of pulling out into a separate commit.